app/controllers/update/RemovePlatformOperatorController.scala

```diff
@@ -70,6 +73,8 @@ class RemovePlatformOperatorController @Inject()(
               cleanedData    = request.userAnswers.copy(data = Json.obj())
               updatedAnswers <- Future.fromTry(cleanedData.set(PlatformOperatorDeletedQuery, businessName))
               _              <- sessionRepository.set(updatedAnswers)
+              responseData = Json.toJsObject(SuccessRemovalResponseData(businessName, operatorId))
+              _ <- auditService.sendAudit[RemovePlatformOperatorAuditEventModel](RemovePlatformOperatorAuditEventModel(responseData).toAuditModel)
```

Comment:
Can it be verified on spec class RemovePlatformOperatorSpec like 
verify(mockAuditService, times(1)).sendAudit(any())
May be few other places might need to do. Please have a quick look if possible.

Reply:
Updated.